doc/api.md

```diff
@@ -49,3 +49,10 @@ curl "http://content-performance-manager.dev.gov.uk/groups/3.json?api_token=1234
 curl "http://content-performance-manager.dev.gov.uk/groups.json?api_token=1234567" \
      -H "Content-Type: application/json"
 ```
+
+* Delete a group
+
+```terminal
+curl "http://content-performance-manager.dev.gov.uk/groups.json?api_token=1234567" \
```

Comment:
That worked but now I have a question about ID vs slug as param which I've asked in https://github.com/alphagov/content-performance-manager/pull/96/commits/6ac84c3c1338bb4236abcf310389304b2c946b82#r107136016